sdk-extension/opentelemetry-sdk-extension-aws/tests/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
Do I add this copyright disclaimer if there's no code here?

Reply:
I would just to be complete. You don't *need* to, to your point.

Reply:
I removed them because an empty file seems simpler and I see lots of cases that have empty files.